src/OpenTracing/ISpan.cs

```diff
@@ -28,6 +29,9 @@ public interface ISpan
         /// <summary>Same as <see cref="SetTag(string,string)"/> but for numeric values.</summary>
         ISpan SetTag(string key, double value);
 
+        /// <summary>Set a tag on the Span using the helper in type.</summary>
```

Comment:
Not sure if it's the fact that English is not my first language but this summary sounds weird. What's the "helper in type"?

Reply:
„Type“ was a reference to the parameter name. Sorry, English isn’t my first language so I didn’t notice it prior. Will adjust it.

Reply:
I renamed it to "tagSetter" and made it a paramref.